modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/ProductOptionValueResourceImpl.java

```diff
@@ -104,9 +104,15 @@ public ProductOptionValue patchProductOptionValue(
 
 		long cpInstanceId = 0;
 
-		CPInstance cpInstance = _cpInstanceService.fetchCProductInstance(
```

Comment:
I think this should not be removed, what id the user don't pass both ERC and SKUId? PATCH should not erase the previous SKUID